beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -102,3 +104,41 @@ func ProcessFinalization(state *pb.BeaconState) *pb.BeaconState {
 	}
 	return state
 }
+
+// ProcessCrosslinks goes through each shard committee and check
+// shard committee's attested balance * 3 was greater than total balance *2.
+// If it's greater then beacon node updates shard committee with
+// the latest state slot and wining root.
+//
+// Spec pseudocode definition:
+//	For every shard_committee in state.shard_committees_at_slots:
+// 		Set state.latest_crosslinks[shard] = CrosslinkRecord(
+// 		slot=state.slot, block_root=winning_root(shard_committee))
+// 		if 3 * total_attesting_balance(shard_committee) >= 2 * total_balance(shard_committee).
+func ProcessCrosslinks(
+	state *pb.BeaconState,
+	shardCommitteesAtSlots []*pb.ShardAndCommitteeArray,
```

Comment:
Can't we obtain this from the state itself via the getter?

Reply:
+1

Reply:
Good call!